docker-compose.yml

```diff
@@ -6,7 +6,7 @@ redis:
   env_file: .env
```

Comment:
Better not to change the .yml because these changes are relevant only for the local deployment, not production version.